src/ext.js

```diff
@@ -5,6 +5,23 @@ export const checkShow = (data, field) => {
   return act && act.requiredInput && act.requiredInput.indexOf(field) !== -1;
 };
 
+export const getValue = (data, reference, defaultValue) => {
+  const steps = reference.split('.');
+  let dataContainer = data;
```

Comment:
Just call the argument `dataContainer`?

Reply:
it's a util function from client originally, but it doesn't seem to make any difference

Reply:
Changing the variable names might make it easier to understand. Also I would suggest breaking this out into another file so we keep ext.js cleaner. (Yes that probably also goes for checkShow and another one that I am adding for navigation :))